scripts/browserstack/start_tunnel.sh

```diff
@@ -1,3 +1,71 @@
-export BROWSER_STACK_ACCESS_KEY=`echo $BROWSER_STACK_ACCESS_KEY | rev`
+#!/bin/bash
 
-node ./scripts/browserstack/start_tunnel.js &
+set -e -o pipefail
+
+# Workaround for Travis CI cookbook https://github.com/travis-ci/travis-ci/issues/4862,
+# where $PATH will be extended with relative paths to the NPM binaries.
+PATH=`echo $PATH | sed -e 's/:\.\/node_modules\/\.bin//'`
+
+TUNNEL_FILE="BrowserStackLocal-linux-x64.zip"
+TUNNEL_URL="https://www.browserstack.com/browserstack-local/$TUNNEL_FILE"
+TUNNEL_DIR="/tmp/browserstack-tunnel"
+TUNNEL_LOG="$LOGS_DIR/browserstack-tunnel.log"
+
+BROWSER_STACK_ACCESS_KEY=`echo $BROWSER_STACK_ACCESS_KEY | rev`
+
+# Cleanup and create the folder structure for the tunnel connector.
+rm -rf $TUNNEL_DIR $BROWSER_PROVIDER_READY_FILE
+mkdir -p $TUNNEL_DIR
+touch $TUNNEL_LOG
+
+cd $TUNNEL_DIR
+
+# Download the saucelabs connect binaries.
```

Comment:
Good catch. I made the two files in the docs setup as similar as possible (so we can share a lot)